packages/components/src/FormField/FormField.tsx

```diff
@@ -150,7 +150,7 @@ export function FormField(props: FormFieldProps) {
               );
             default:
               return (
-                <>
+                <div style={{ position: "relative" }}>
```

Comment:
what's this one for?

Reply:
I meant to move this into a style rather than doing it inline, so it shouldn't be here but as for why it's there at all:

https://github.com/GetJobber/atlantis/pull/1856#issuecomment-2038391680

Reply:
we no longer need this because we're not trying to place the toolbar in the same container as the input, and so we can rely on the `relative` provided by the inputWrapper I believe